docs/editors-guide/merging-and-obsoleting.md

```diff
@@ -89,51 +98,63 @@ mv new.obo mondo-edit.obo
-1. Add an obsoletion reason: use the annotation property 'has obsolescence reason' and write 'terms merged' in the literal field.
-1. Any source annotations to MONDO:equivalentTo on a dbxref should be changed to MONDO:obsoleteEquivalent.
-1. _Optional:_ Add an additional comment (rdfs:comment) explaning why the terms were merged.
+    1. Add "term tracker item" (type xsd:anyURI) annotation with a link to the GitHub issue that requested the obsoletion.
+    1. Add an obsoletion reason: use the annotation property "has obsolescence reason" and add `http://purl.obolibrary.org/obo/MONDO_TermsMerged` in the IRI Editor field.
+    1. Any source annotations to MONDO:equivalentTo on a dbxref should be changed to MONDO:obsoleteEquivalent.
+    1. _Optional:_ Add an additional comment (rdfs:comment) explaning why the terms were merged.
 1. **Replacement term**: Search for the 'term replaced by' term
-1. Delete the old ID (_if you check the diff after doing this and saving, you'll see the unexpected addition will disappear_)
-1. Review the annotations to ensure there are no duplicate annotations. If there are, they should be combined. (Sometimes this is easier to review in the diff in GitHub Desktop, but they must be fixed in Protege.)
-1. Review the subClassOf assertions, and make sure there are no duplicates. If there are, they should be combined.
-1. Remove the annotations: 'in subset obsoletion candidate' and 'scheduled for obsoletion on or after'  
-1. When reviewing the diff, make sure there is not an Alt ID. The diff should only show additions to the merged term and the obsoletion
+    1. Delete the old ID (_if you check the diff after doing this and saving, you'll see the unexpected addition will disappear_)
+    1. Review the annotations to ensure there are no duplicate annotations. If there are, they should be combined. (Sometimes this is easier to review in the diff in GitHub Desktop, but they must be fixed in Protege.)
+    1. Review the subClassOf assertions, and make sure there are no duplicates. If there are, they should be combined. This can be done in a text editor by copying the "source" information from one subClassOf assertion to the other and deleting the duplicate subClassOf assertion. After saving the file, the updates will be visible in Protege.
+    1. Remove the annotations:
+        1. "in subset obsoletion candidate"
+        1. "scheduled for obsoletion on or after"  
+    1. When reviewing the diff, make sure there is not an Alt ID. The diff should only show additions to the merged term and the obsoleted class.
+
+    ---
+    <h3>Merge Terms in Mondo - Part 1</h3>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/hhuq4Lwz0Ds" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
is this working? It doesn't appear to display correctly when I view the file

Reply:
Yes, it is for me.